CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # Change Log
+## 1.25.8 - 2022-03-08
+### Changed
+* Fixed dev container load errors when dependency list include VSCode Remote Containers extension.

```

Comment:
>Fixed dev container load erro [](http://example.com/codeflow?start=2&length=29)

I think the line should be same as the readme

Reply:
Thank you!